### PR TITLE
Centralize all TestNet specialization behind a switch

### PR DESCRIPTION
**Pull request by Geod24**
```
This should have been done this way in the first place.
The realm in Config was left to TestNet but could be changed back anytime.
```

CC @mkykadir

### CODE REVIEW

File: devel/testnet/config.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@
 node:
   realm: "testnet.bosagora.io"
   testing: true
-  data_dir: .fullnode/data/
+  data_dir: data/
```
Comment (mkykadir) [inline devel/testnet/config.yaml:16]: 👍

**Review by mkykadir — APPROVED** [review]

File: devel/testnet/config.yaml (hunk 1 above)
Comment (Geod24) [reply devel/testnet/config.yaml:16]: Will apply to `--testnet` as well

**Comment by Geod24** [comment]
Fun fact: If we go with modifying `dub.json` to have extra string import path, we could do:
```
parseString(mixin("devel/testnet/config.yaml"));
```

And the configuration file would be embedded into Agora :D

**Comment by Geod24** [comment]
Updated

**Pull request merged**